packages/flutter_tools/test/general.shard/project_test.dart

```diff
@@ -185,6 +185,13 @@ void main() {
         await project.regeneratePlatformSpecificTooling();
         expect(testLogger.statusText, contains('https://flutter.dev/go/android-project-migration'));
       });
+      _testInMemory('Android plugin without example app does not show a warning', () async {
+        final FlutterProject project = await aPluginProject();
+        project.example.directory.deleteSync();
+
+        await project.regeneratePlatformSpecificTooling();
+        expect(testLogger.statusText, isNot(contains('https://flutter.dev/go/android-project-migration')));
```

Comment:
we're gonna forget to update this test if that link changes. I don't have a better idea though

Reply:
Nah we'll get it, the tests that are checking for this URL will fail.